tst/testinstall/opers/StructureDescription.tst

```diff
@@ -0,0 +1,69 @@
+gap> START_TEST("StructureDescription.tst");
+gap> l := AllSmallGroups(12);;
+gap> List(l, StructureDescription);; l;
+[ C3 : C4, C12, A4, D12, C6 x C2 ]
+gap> List(AllSmallGroups(40),G->StructureDescription(G:short));
+[ "5:8", "40", "5:8", "5:Q8", "4xD10", "D40", "2x(5:4)", "5:D8", "20x2", 
+  "5xD8", "5xQ8", "2x(5:4)", "2^2xD10", "10x2^2" ]
+gap> List(AllSmallGroups(16), G -> StructureDescription(G:recompute));
+[ "C16", "C4 x C4", "(C4 x C2) : C2", "C4 : C4", "C8 x C2", "C8 : C2", "D16", 
+  "QD16", "Q16", "C4 x C2 x C2", "C2 x D8", "C2 x Q8", "(C4 x C2) : C2", 
+  "C2 x C2 x C2 x C2" ]
+gap> List(AllSmallGroups(60), G -> StructureDescription(G:recompute));
+[ "C5 x (C3 : C4)", "C3 x (C5 : C4)", "C3 : (C5 : C4)", "C60", "A5", 
+  "C3 x (C5 : C4)", "C3 : (C5 : C4)", "S3 x D10", "C5 x A4", "C6 x D10", 
+  "C10 x S3", "D60", "C30 x C2" ]
+gap> List(AllPrimitiveGroups(DegreeAction, 8), StructureDescription);
+[ "(C2 x C2 x C2) : C7", "(C2 x C2 x C2) : (C7 : C3)", 
+  "(C2 x C2 x C2) : PSL(3,2)", "PSL(3,2)", "PSL(3,2) : C2", "A8", "S8" ]
+gap> List(AllTransitiveGroups(DegreeAction, 6), G -> StructureDescription(G:short));
+[ "6", "S3", "D12", "A4", "3xS3", "2xA4", "S4", "S4", "S3xS3", "(3^2):4", 
+  "2xS4", "A5", "(3^2):D8", "S5", "A6", "S6" ]
+gap> StructureDescription(PSL(4,2));
+"A8"
+gap> G := Group([ (4,8)(6,10), (4,6,10,8,12), (2,4,12)(6,10,8), (3,9)(4,6,10,8,12)(7,11), (3,5)(4,6,10,8,12)(9,11), (1,3,11,9,5)(4,6,10,8,12) ]);;
+gap> StructureDescription(G);
+"A5 x A5"
+
+##
+## gap> G := Group([ (6,7,8,9,10), (8,9,10), (1,2)(6,7), (1,2,3,4,5)(6,7,8,9,10) ]);;
+## gap> StructureDescription(G);
+## "A5 : S5"
+##
```

Comment:
Yes, there is. Currently this example works. After merging this PR, this example will not work. The reason is that the new ````SemidirectDecompositions```` code uses ````ComplementClassesRepresentatives```` which does not work if both N and G/N is non-solvable. A proposed solution and a discussion about this arose in my PR #563.